src/commands/apps.rs

```diff
@@ -68,6 +78,29 @@ impl AppsCommand {
                     .with_context(|| format!("Problem deleting app named {}", &cmd.app))?;
                 println!("Deleted app \"{}\" successfully.", &cmd.app);
             }
+            AppsCommand::Get(cmd) => {
+                let (client, app_id) =
+                    client_and_app_id(cmd.common.deployment_env_id.as_deref(), &cmd.app).await?;
+                let app = client
+                    .get_app(app_id.to_string())
+                    .await
+                    .with_context(|| format!("Error: could not get details about {}", &cmd.app))?;
+
+                println!("Name: {}", &app.name);
+                println!("Description: {}", &app.description.unwrap_or_default());
+                println!(
+                    "URL: https://{}",
+                    app.channels[0].domain.as_ref().unwrap_or(&String::new())
+                );
+                if let Some(val) = app.domain {
+                    let status = match val.validation_status {
+                        cloud_openapi::models::DomainValidationStatus::InProgress => "In progress",
```

Comment:
nit: I would import `use cloud_openapi::models:DomainValidationStatus` instead of using the fully qualified path here
```suggestion
                        DomainValidationStatus::InProgress => "In progress",
```

Reply:
Done